packages/lodestar/src/sync/reqResp/reqResp.ts

```diff
@@ -218,7 +218,7 @@ export class BeaconReqRespHandler implements IReqRespHandler {
     peerId: PeerId,
     sink: Sink<unknown, unknown>
   ): Promise<void> {
-    this.logger.info(`Received goodbye request from ${peerId.toB58String()}, reason=${request}`);
+    this.logger.info(`Received goodbye request from ${peerId.toB58String()}, reason=${request.body}`);
```

Comment:
Note that here you are still stringifying the data instead of passing JSON to the logger. I would suggest to do this:
```ts
this.logger.info("Received goodbye request", {peer: peerId.toB58String(), reason: request.body});
```
If you follow this suggestion please test at least once that the output looks as expected

Reply:
ah yeah, the logger doesn't like bigints apparently.  when i try your line i get 
```Types of property 'reason' are incompatible.
    Type 'bigint' is not assignable to type 'string | number | boolean | { [property: string]: Json; } | Json[] | null | undefined'.```
in my IDE

Reply:
Then we should modify the logger to accept bigints. Do a similar approach like describe here https://github.com/ChainSafe/lodestar/pull/1755#issuecomment-726914113
Add test cases for the data that needs to be logged and modify the logger functions accordingly